environments/common/application-load-balancer/alb.tf

```diff
@@ -43,9 +43,14 @@ resource "aws_lb_listener" "trade_tariff_listeners" {
   ssl_policy        = "ELBSecurityPolicy-TLS-1-2-2017-01"
   certificate_arn   = var.certificate_arn
 
+  # our rules will take over
   default_action {
-    type             = "forward"
-    target_group_arn = aws_lb_target_group.trade_tariff_target_groups["frontend"].arn
+    type = "fixed-response"
+    fixed_response {
+      content_type = "text/plain"
+      message_body = "OK"
+      status_code  = "200"
```

Comment:
Should this be 404?

Reply:
That's much better behaviour - I'll push a commit that returns a 404. 🙇🏻‍♀️